src/handlers/health.js

```diff
@@ -38,7 +38,7 @@ module.exports = {
    */
   get: async (request, h) => {
     try {
-      let {response, statusCode} = await handler.getHealth()
+      const { statusCode } = await handler.getHealth()
```

Comment:
You can replace this with the `central-services-health` library along with the shared HealthCheck implementation:

```js
const HealthCheck = require('@mojaloop/central-services-shared').HealthCheck.HealthCheck
const { defaultHealthHandler } = require('@mojaloop/central-services-health')

const healthCheck = new HealthCheck(packageJson, [])

module.exports = {
  get: defaultHealthHandler(healthCheck)
}

```